src/Sarif/FileRegionsCache.cs

```diff
@@ -92,6 +105,150 @@ public void ClearCache()
             this._cache.Clear();
         }
 
+        public Region SuvamTest(
```

Comment:
This is a placeholder public function for testing purposes, and will go away in a subsequent commit.